DigitalLearningSolutions.Data/Enums/AdminCreationError.cs

```diff
@@ -6,9 +6,9 @@ public class AdminCreationError : Enumeration
             1,
             nameof(UnexpectedError)
         );
-        public static AdminCreationError EmailAlreadyInUse = new AdminCreationError(
+        public static AdminCreationError ActiveAdminAlreadyExists = new AdminCreationError(

```

Comment:
Could change to `AlreadyActiveAdminAtCentre` just to be a little more specific, as having Active Admins shouldn't prevent this in all cases.